tests/x11/ghostscript.pm

```diff
@@ -31,8 +31,8 @@ use testapi;
 use utils;
 
 sub run {
-    select_console "x11";
-    x11_start_program('xterm');
+    my $self = shift;
+    $self->select_serial_terminal;
 
     # become root, disable packagekit and install all needed packages
     become_root;
```

Comment:
`select_serial_terminal()` gives you a root terminal unless you specifically ask for unprivileged user.